sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipeline.java

```diff
@@ -137,10 +144,17 @@ private void onResponseBody(
             return;
         }
 
-        listener.onResponse(request, new TelemetryPipelineResponse(responseCode, responseBody));
+        TelemetryPipelineResponse telemetryPipelineResponse = new TelemetryPipelineResponse(responseCode, responseBody);
+        listener.onResponse(request, telemetryPipelineResponse);
         if (responseCode == 200) {
             result.succeed();
         } else {
+            if (responseCode == 400
+                && statsbeatModule != null
+                && telemetryPipelineResponse.isInvalidInstrumentationKey()) {
+                LOGGER.warning("400 status code is returned for an invalid customer's instrumentation key. Shutting down Statsbeat.");
```

Comment:
in general we don't want to log any warnings or errors about statsbeat

```suggestion
                LOGGER.debug("400 status code is returned for an invalid instrumentation key. Shutting down Statsbeat.");
```

Reply:
(this doesn't need to be done in this PR, I don't think very likely for it to cause us problems the way prior statsbeat warnings did)

Reply:
this is clientlogger, doesn't have debug logging level. i can change it to Verbose?

Reply:
👍